shells/dev-shell/store-panel.js

```diff
@@ -125,7 +125,13 @@ export class StorePanel extends HTMLElement {
   async attach(store) {
     this.store = store;
     this.storeName.textContent = store.name || store.id;
-    this.schema.textContent = this.store.backingType().entitySchema.toManifestString();
+    const schema = this.store.backingType().entitySchema;
+    if (schema) {
+      this.schema.textContent = schema.toManifestString();
+    } else {
+      console.log(this.store.backingType());
```

Comment:
Did you mean to leave this in? If so, might be useful to include some descriptive text.

Reply:
Ah, no, I'd been looking at getting the type information out but I don't think its worth doing now.

Will remove before merge.